_posts/2017-08-27-Clean-Architecture.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+author: kris
+header-img: "img/architecture-bg.jpg"
```

Comment:
background foto laad wel traag, ook al is het maar 700k. misschien moeten we onze fotootjes nog kleiner maken?

Reply:
Je kan altijd voor een cdn gaan lost vaak het probleem op